packages/forms/resources/css/components/rich-editor.css

```diff
@@ -22,3 +22,15 @@
 trix-toolbar .filament-forms-rich-editor-component-toolbar-button.trix-active {
     @apply bg-primary-600 text-white;
 }
+
+[dir='rtl'] .prose :where(ul):not(:where([class~='not-prose'] *)),
```

Comment:
@danharrin, this CSS isn't scoped to the rich-editor in any way... sure this won't cause issues in other places?

Reply:
should be fixed

Reply:
@danharrin, could you revert this PR?
Then @aymanalareqi can attempt a new PR with a correct fix?

Reply:
Never mind, I didn't read your "should be fixed" as "has been fixed". 😄